plugins/CoreHome/angularjs/enrichedheadline/enrichedheadline.directive.html

```diff
@@ -13,18 +13,21 @@
            title="{{ 'CoreHome_ExternalHelp'|translate }}"
            class="helpIcon"><span class="icon-help"></span></a>
 
-        <a ng-if="inlineHelp"
```

Comment:
just a minor point but I think something like this might introduce less repeated code:
`<a ng-if="inlineHelp"
           title="{{ (reportGenerated ? 'General_HelpReport' : 'General_Help' )|translate }}"`